src/client/datascience/jupyter/kernels/kernel.ts

```diff
@@ -611,7 +609,10 @@ export class Kernel implements IKernel {
     }
 }
 
-export async function executeSilently(session: IJupyterSession, code: string): Promise<nbformat.IOutput[]> {
+export async function executeSilently(
+    session: Pick<JupyterKernel.IKernelConnection, 'requestExecute'>,
```

Comment:
No longer using our types, directly using Jupyter Lab API types.
Much easier to refactor later on (as we have fewer layers)